templates/gateway/ingress.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           {{- else }}
           - path: {{ $.Values.graphdb.workbench.subpath }}(/|$)(.*)
           {{- end }}
-            pathType: Prefix
+            pathType: ImplementationSpecific
```

Comment:
shouldn't we make this a variable?

Reply:
No, if you use regex in the path, you should be using this path type. 

>  path /graphdb(/|$)(.*) cannot be used with pathType Prefix